lib/_tls_wrap.js

```diff
@@ -242,6 +247,55 @@ function onnewsession(key, session) {
     done();
 }
 
+function onPskServerCallback(identity, maxPskLen) {
+  const owner = this[owner_symbol];
+  const psk = owner[kPskCallback](identity);
+  if (psk == null) {
+    return undefined;
+  } else {
+    if (!isArrayBufferView(psk)) {
+      throw new ERR_INVALID_ARG_TYPE(
+        'psk',
+        ['Buffer', 'TypedArray', 'DataView'],
+        psk
+      );
+    }
+    if (psk.length > maxPskLen) {
+      throw new ERR_ASSERTION(`Pre-shared key exceeds ${maxPskLen} bytes`);
```

Comment:
I believe this err is specifically to be used by `assert()`, in which case, you could call assert. However, I think one of these errors would be better:
- ERR_OUT_OF_RANGE; or possibly
- ERR_INVALID_ARG_VALUE

@addaleax @Trott you are more familiar with current error conventions, do you agree?

Reply:
I'll change it to `ERR_INVALID_ARG_VALUE` for now.